src/Chapter1.hs

```diff
@@ -640,9 +651,13 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
-
+firstDigit :: Int -> Int
+firstDigit n 
+  | reduceN n == 0 = n
```

Comment:
Yes, this is right!
But, needs a small patch in order to work on negative numbers 🙂 
At the moment it will return `-5` on the `-5` input, though it should be `5`.

Reply:
Thanks a  lot, now I could co-relate those make scripts hanging :)